test/TransitivePathTest.cpp

```diff
@@ -130,20 +104,19 @@ TEST_P(TransitivePathTest, idToVar) {
                       left, right, 1, std::numeric_limits<size_t>::max());
 
   auto resultTable = T->computeResultOnlyForTesting();
-  assertSameUnorderedContent(expected, resultTable.idTable());
+  ASSERT_THAT(resultTable.idTable(),
+              ::testing::UnorderedElementsAreArray(expected));
 }
 
 TEST_P(TransitivePathTest, varToId) {
-  IdTable sub(2, makeAllocator());
-  sub.push_back({V(0), V(1)});
-  sub.push_back({V(1), V(2)});
-  sub.push_back({V(1), V(3)});
-  sub.push_back({V(2), V(3)});
+  auto sub = makeIdTableFromVector(
+      {{V(0), V(1)}, {V(1), V(2)}, {V(1), V(3)}, {V(2), V(3)}});
 
-  IdTable expected(2, makeAllocator());
-  expected.push_back({V(2), V(3)});
-  expected.push_back({V(1), V(3)});
-  expected.push_back({V(0), V(3)});
+  auto expected = makeIdTableFromVector({
+      {V(2), V(3)},
+      {V(1), V(3)},
+      {V(0), V(3)},
```

Comment:
If you want to, you can even drop the `V(...)`, so just
`makeIdTableFromVector({{2,3}, {1,3}, {0, 3}})`. I think I have an overload for just such conversions where we do not really care about the contents of IDs if they are the same.